components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -321,6 +323,7 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
             const includeUsers = Object.values(this.props.includeUsers);
             users = [...users, ...includeUsers];
         }
+        users = [...filterProfilesStartingWithTerm(this.props.profilesFromRecentDMs, this.state.term).slice(0, USERS_FROM_DMS).map((user) => user as UserProfileValue), ...users].slice(0, MAX_USERS).filter(((profile, index, profiles) => profiles.indexOf(profile) === index));
```

Comment:
I've got a few comments here:
1. Do recent users need to be filtered out like we're doing with `profilesNotInCurrentChannel` above? Perhaps we could wrap these steps in a function so that we can apply them to both sets of users. It looks like we're filtering those to ensure that:

    * Deleted users aren't shown
    * Users that aren't members of the current team aren't shown
    * Excluded users aren't shown
    
2. Do we need to do anything to dedupe users for anyone who shows up in both users and in `profilesFromRecentDMs`?

3. Finally, would you mind splitting this up across a few lines? It's a bit hard to read where different sets of brackets begin and end here

Reply:
1. created a small function; 
2. I think Dedupe is necessary, otherwise, We'll see one username twice.
3. Sure, I'll split the code into multiple lines.